wpiutil/CMakeLists.txt

```diff
@@ -147,7 +149,12 @@ if (MSVC)
     target_compile_definitions(wpiutil PRIVATE -D_CRT_SECURE_NO_WARNINGS)
 endif()
 wpilib_target_warnings(wpiutil)
-target_link_libraries(wpiutil Threads::Threads ${CMAKE_DL_LIBS} ${ATOMIC})
+target_link_libraries(wpiutil Threads::Threads ${CMAKE_DL_LIBS})
+
+if (ATOMIC)
+    target_link_libraries(wpiutil ${ATOMIC})
+endif()
```

Comment:
Why is it necessary to do this?  ATOMIC will just expand to nothing in the old code if it's not set...

Reply:
No, it expands to ATOMIC_NOTFOUND